data/scenarios/Testing/1575-structure-recognizer/1575-handle-overlapping.yaml

```diff
@@ -0,0 +1,76 @@
+version: 1
+name: Structure recognizer - Overlaps
+description: |
+  Completing a row that lies between two partially-complete structures
+  may complete both of them, but only one will be registered.
```

Comment:
I settled on mutually-exclusive structure membership for each cell by the rationale that 70 structure instances is "too many", in the [example you've given](https://github.com/swarm-game/swarm/pull/1579#discussion_r1376365696).

> there is no way to control which of the two possible structures is the one that will actually be registered.

It will in part be up to the scenario author not to define structure templates that are conducive to overlap, or to minimize the significance of exact positioning when overlaps happen.